content/docs/04-clusters/04-palette-virtual-clusters/02-deploy-virtual-cluster.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Deploy a Virtual Cluster to a Cluster Group"
+metaTitle: "Deploy a Virtual Clusters to a Cluster Group"
+metaDescription: "How to add Palette Virtual Clusters to a Cluster Group"
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Add Virtual Clusters to a Cluster Group
+
+
+You can deploy Palette Virtual Clusters to a [cluster group](/clusters/cluster-groups). The advantages of a virtual cluster environment are:
+- You can operate with admin-level privileges while ensuring strong isolation.
+- Virtual clusters reduce operational overhead and improve resource utilization.
+
+Use the followings steps to deploy a virtual cluster.
+
+# Prerequisites
+
+- A Spectro Cloud account.
+
+- A cluster group. Refer to the [Create and Manage Cluster Groups](/clusters/cluster-groups/create-cluster-group) guide to learn how to create a cluster group.
+
+- Attach any required policies in your cloud account that must be added to your virtual cluster deployment. 
+  - For AWS, refer to the [Required IAM Policies](/clusters/public-cloud/aws/required-iam-policies#globalroleadditionalpolicies) documentation.
+  - For Azure, no additional policies are required.
+
+<InfoBox>
+
+Palette doesn't support _Usage_ and _Cost_ metrics for Virtual Clusters running on Google Kubernetes Engine (GKE).
+
+</InfoBox>
+
+## Add Node-Level Policies in your Cloud Account
+
+In some situations additional node-level policies must be added to your deployment. 
+
+To add node-level policies: 
+
+1. In **Cluster Mode**, switch to the **Tenant Admin**  project.
+
+
+2. Select **Tenant Settings** in the **Main Menu**. 
+
+
+3. Click **Cloud Accounts** and ensure **Add IAM policies** is enabled for your cloud account. If an account does not already exist, you must add one.
+
+
+4. You can specify any additional policies to include in virtual clusters deployed with this cloud account.
+
+    - For AWS, add the **AmazonEBSCSIDriver** policy so that the virtual clusters can access the underlying host cluster's storage. Check out the [Palette required IAM policies](/clusters/public-cloud/aws/required-iam-policies#globalroleadditionalpolicies) documentation to learn more about additional IAM policies.
+
+
+5. Confirm your changes.
+
+# Deploy a Virtual Cluster
+
+Follow these steps to deploy a virtual clusters to a cluster group:
+
+1. Log in to [Palette](https://console.spectrocloud.com) and switch to **App Mode**.
+
+
+2. Select **Virtual Clusters** from the left **Main Menu**. 
+
+
+3. Click on the **+New Virtual Cluster**.
```

Comment:
```suggestion
3. Click the **+New Virtual Cluster**.
```

Reply:
Are we including the + in the button name? I've been using only the text label. As a user, I see the + as a visual indicator that I can add something, but to call it by name I'd say **New Virtual Cluster** button or click **New Virtual Cluster.** Maybe we need to discuss and add to the internal style guidelines.

Reply:
Yeah, let's chat. I'll add it to the team agenda